vk-vtlayer-generate.py

```diff
@@ -1174,16 +1174,17 @@ def generate_intercept(self, proto, qual):
                      '    PFN_vkGetInstanceProcAddr fpGetInstanceProcAddr = chain_info->u.pLayerInfo->pfnNextGetInstanceProcAddr;\n'
                      '    PFN_vkCreateInstance fpCreateInstance = (PFN_vkCreateInstance) fpGetInstanceProcAddr(NULL, "vkCreateInstance");\n'
                      '    if (fpCreateInstance == NULL) {\n'
-                     '                return VK_ERROR_INITIALIZATION_FAILED;\n'
+                     '        (*outputStream) << "t{" << getTIDIndex() << "} vkCreateInstance FAILED TO FIND PROC ADDRESS" << endl;\n'
+                     '        return VK_ERROR_INITIALIZATION_FAILED;\n'
                      '    }\n'
                      '    // Advance the link info for the next element on the chain\n'
                      '    assert(chain_info->u.pLayerInfo);\n'
                      '    chain_info->u.pLayerInfo = chain_info->u.pLayerInfo->pNext;\n'
                      '    VkResult result = fpCreateInstance(pCreateInfo, pAllocator, pInstance);\n'
-                     '    if (result != VK_SUCCESS)\n'
-                     '        return result;\n'
-                     '    initInstanceTable(*pInstance, fpGetInstanceProcAddr);\n'
-                     '    createInstanceRegisterExtensions(pCreateInfo, *pInstance);\n'
+                     '    if (result == VK_SUCCESS) {\n'
+                     '        initInstanceTable(*pInstance, fpGetInstanceProcAddr);\n'
+                     '        createInstanceRegisterExtensions(pCreateInfo, *pInstance);\n'
+                     '    }\n'
                      '    %s%s%s\n'
```

Comment:
My first comment on this code is by not returning early when result != VK_SUCCESS, then api_dump will try to print out all the parameters.  If the cause of the failure was bad parameters then api_dump may cause a crash before printing anything out.  I think this is okay since the loader and ICD also may crash with bad parameters  to CreateInstance.

By skipping the dispatch table initialization you are ensuring that any subsequent Vulkan call will likely crash. 

Overall LGTM.